### PR TITLE
Fix DisGeNET web VEP

### PR DESCRIPTION
**Pull request by dglemos**
PR to fix DisGeNET for release 102 
Tickets: ENSVAR-3390 and ENSVAR-2605

### CODE REVIEW

**Comment by dglemos** [comment]
Original PR #364

File: tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm [hunk 1]
```diff
@@ -1139,11 +1151,17 @@ sub get_items_in_list {
...
+      $entry =~ s/_/ /g;
+      push (@items_with_url, $entry);
+    }
+  }
   # Add external links
   else {
     foreach my $item (@items_list) {
       my $item_url = $item;
-      if ($type eq 'pubmed') {
+      if ($type eq 'pubmed' || $type eq 'disgenet_pmid') {
```
Comment (helensch) [inline tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm:1164]: For example rs797044835, the digenet_pmid are not links. Should they be?

**Review by helensch — COMMENTED** [review]
Looks ok. Minor queries re url and links 
Suggest tooltip on web results form for "DisGeNET SCORE" use DisGeNET instead of DisGENET.
The images were not checked as are different to those on sandbox i.e.  input form docs do not show disgenet option in image (vep_web_annotation.png)

File: tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm (hunk 1 above)
Comment (dglemos) [reply tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm:1164]: They should be. Can you check again? The branch was wrong.

File: docs/htdocs/info/docs/tools/vep/online/input.html [hunk 2]
```diff
@@ -323,6 +323,14 @@ <h2 id="annotation">Additional annotations</h2>
                     <b>Note:</b> This web functionality is not reporting cancer phenotypic data this release. However the cancer phenotypic data is available in the command line version.
                   </p>
                 </li>
+
+                <li><b>DisGeNET</b>
+                  <p>
+                    Report Variant-Disease-PMID associations from the <a href="http://www.disgenet.org/" rel="external">DisGeNET</a> database. This functionality is provided by the
```
[the remark this answers is not in the document]
Comment (dglemos) [reply docs/htdocs/info/docs/tools/vep/online/input.html]: I fixed it. Also, I opened a PR in VEP_plugins to fix the URL in the helptip.

**Comment by dglemos** [comment]
@helensch thank you for the comments. The issues should be fixed now.

File: tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm (hunk 1 above)
Comment (helensch) [reply tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm:1164]: links now working.

File: docs/htdocs/info/docs/tools/vep/online/input.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (helensch) [reply docs/htdocs/info/docs/tools/vep/online/input.html]: Thanks

**Review by helensch — CHANGES_REQUESTED** [review]
Images will be reviewed by @aparton, otherwise PR ok

**Review by aparton — APPROVED** [review]

**Review by aparton — CHANGES_REQUESTED** [review]
Images are ok if not ideal, new images incoming

**Comment by dglemos** [comment]
I'm going to make a final update to images: input, annotation and prediction.

**Review by aparton — APPROVED** [review]

**Comment by aparton** [comment]
Not ideal, but better than they were. I'm happy to merge

**Review by helensch — APPROVED** [review]

**Comment by helensch** [comment]
@aparton thanks for reviewing the images

**Pull request merged**